app/assets/javascripts/details.js

```diff
@@ -0,0 +1,25 @@
+
+function addDynamicSummaryTextForExpandedAndCollapsedDetailsTag(detailsElement) {
+    var expandedText = detailsElement.getAttribute('data-summary-expanded');
+    var collapsedText = detailsElement.getAttribute('data-summary-collapsed');
+    var summaryTextElement = detailsElement.getElementsByClassName('govuk-details__summary-text').item(0);
+
+    if(summaryTextElement && collapsedText && expandedText) {
```

Comment:
Could you maybe use extract method refactoring to make this condition clearer?

Reply:
Or maybe just refactor it to a named condition like `var myNamedCondition = summaryTextElement && collapsedText && expandedText` so that it reads as `if(myNamedCondition)`

Reply:
I've extracted a variable, let me know if you think it's clearer.

Reply:
Yep, it's clearer now 👍